package.json

```diff
@@ -34,7 +30,7 @@
   },
   "dependencies": {
     "@acuris/aws-es-connection": "^1.1.0",
-    "@elastic/elasticsearch": "^7.10",
+    "@elastic/elasticsearch": "^7.9.0",
```

Comment:
what was the problem here with 7.10?

Reply:
The `@acuris/aws-es-connection` package only supports `@elastic/elasticsearch` up to version 7.9.0.

https://github.com/mergermarket/acuris-aws-es-connection/blob/3334c73a96db731970933c3240f1059c41301e50/package.json#L36

The 2.x.x version of `@acuris/aws-es-connection` supports later versions, but that contains breaking changes. For now we should be okay using 7.9

Reply:
ah, good to know